src/components/Avatar/Avatar.jsx

```diff
@@ -178,7 +178,7 @@ const AvatarImage = styled.div(
     cursor: pointer;
     width: ${size};
     height: ${size};
-    border-radius: ${variant ? 2 : '50%'};
+    border-radius: ${variant ? 4 : '50%'};
```

Comment:
Aqui não funciona passar so o valor 4, teria que adicionar o "px" para funcionar, mas é melhor usar o space direto do theme, então na linha 177 utilizar depois do variant `theme: { space }` e utiliza ele para chamar o valor 4 direto do theme.
```suggestion
    border-radius: ${variant ?  `${space[2]}px` : '50%'};
```